web-platform-when-why/index.html

```diff
@@ -0,0 +1,423 @@
+            </aside>
+          </section>
+
+          <section>
+            <p>
+              The Web <strong>is not</strong>&hellip;
+            </p>
+            <ul>
+              <li class="fragment">An operating system</li>
+              <li class="fragment">A hardware specification</li>
+            </ul>
+            <aside class="notes">
+              First, the Web is not an operating system nor a hardware specification,
+              the Web is unique in nature.
+            </aside>
+          </section>
+
+          <section>
+            <p>
+              The Web <strong>is</strong>&hellip;
+            </p>
+            <ul>
+              <li class="fragment">An unprecedent effort of <strong>consensus</strong></li>
+              <li class="fragment">A virtual machine for HTML5</li>
+              <li class="fragment">A <strong title="As in free speech">free</strong> model of content distribution</li>
+            </ul>
+            <aside class="notes">
+              Second, the web is an unprecedent effort of consensus which leads to third,
+              the goals of this agreement are two: interoperability &mdash;as webpages are running
+              in an standarized virtual machine with several implementations (Web engines);
+              and a free &mdash;as in free speech&mdash; model of content distributions.
+            </aside>
+          </section>
+
+          <section>
+            <h3>Interoperability</h3>
+            <img alt="Unique visitors milestones" src="./imgs/unique-visitors-milestones.png"/>
+            <aside class="notes">
+              About interoperability, note that, according to the U.S. report about Mobile Opportunities
+              of 2015, mobile Web usage is increasing without affecting desktop usage. The opportunity is
+              not shifting the focus but going multiplatform. HTML5 is already universal. It is
+              multiplatform for free.
+            </aside>
+          </section>
+
+          <section>
+            <blockquote>
+              <p>Confluence of factors is driving the huge uptick in app usage time, and apps are overtaking other media in importance. However, it’s also important to recognize the others haven’t gone away and the opportunity lies in multi-platform engagement.</p>
+              <p style="text-align: right;">&mdash;2015 U.S. Mobile Report Opportunities Key Points</p>
+            </blockquote>
+          </section>
+
+          <section>
+            <h3>Lack of friction</h3>
+            <img alt="Every step costs you 20% of users" src="./imgs/step-cost-20.png" style="width: 70%;"/>
+            <aside class="notes">
+              The other benefit is actually the great differentiator. The Web spreads through the Internet
+              which is not a propietary marketplace but a completely free information network. The URL is
+              viral, anyplace in the Internet can point you to any Web application without never facing an
```

Comment:
"any place in the Internet", "without **ever**" (without has an implicit denial)